packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -39,18 +39,28 @@ describe('<RadioGroup />', () => {
 
   it('should support uncontrolled mode', () => {
     const wrapper = shallow(
-      <RadioGroup name="group">
+      <RadioGroup name="group" defaultValue="zero">
```

Comment:
I wouldn't modify this unit test; I think adding another one would be the best here, something along the lines of 'uncontrolled mode should support default value'.

Reply:
done